### DISCUSSION TITLE
Help needed - how to erase an old Tasmota installation?

### DISCUSSION THREAD

**Discussion by mstreng411** [Blackymas/NSPanel_HA_Blueprint#1470] [Q&A] [answered]
I have several nspanels with this blueprint in my house and i really like it.
Today i tried to flash my oldest nspanel on which was a Tasmota installation from blakadder running.
The flashing process went fine, but the UI is not coming up. 
And now in HA i have a hybrid, half Tasmota and half ESPHome.
![nspanel](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/72731102/1ed6f3a9-58d5-4c27-8673-2f5080703fda)
I think i have to delete the Tasmota stuff, but i don't know how.

Any suggestions?
Thanks in advance...

**Comment by edwardtfn** [comment]
Sorry, I'm not an expert with Tasmota. I believe you can just remove/delete this specific device from that integration and restart Home Assistant and you shoukd be good.
However, that shouldn't be the root problem here. If you don't have your panel working it can be something else and here I probably can help with.
What can you see in the panel's screen? Can you send a picture?

**Comment by mstreng411** [comment]
The display only shows the bootscreen:
![nsp1](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/72731102/d7ada9ed-cac6-4dbf-840d-35ddf709cef9)
But in Home Assistant all entities are available.

**Reply by edwardtfn** [reply]
Do you have the Blueprint set in an automation? Is the Blueprint also v4.1.x and is the panel selected there?

**Reply by edwardtfn** [reply]
So, please send me the trace following these steps:
1. Go to Developer Tools, tab YAML and click to reload Automations.
2. Go to Settings > Automation and Scenes and click on the automation related to this panel (the one using the Blueprint)
3. Click the 3-dot menu in the top and select "Traces" (if you are in a computer it may be directly as a tab).<br>![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/b066120e-7f2a-4c44-8bd2-8a5bda4d5291)
4. Click to download the trace.<br>![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/c85d17bf-b960-4966-90fc-f8d7608242a5)
5. Paste here the content of that downloaded trace.

**Comment by mstreng411** [comment]
Yes, the blueprint is v4.1.4
The content of the trace is in the attached file.
Thank you for your support!
[trace automation.nspanel1_configuration 2023-12-24T14_37_50.410592+00_00.json](https://github.com/Blackymas/NSPanel_HA_Blueprint/files/13762129/trace.automation.nspanel1_configuration.2023-12-24T14_37_50.410592%2B00_00.json)

**Reply by edwardtfn** [reply, marked as answer]
`"nspanel_deviceid": [],`

You have to open the automation and select your NSPanel on the very first field.

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/fb5b0f20-a3b0-43ae-b3bd-eab930ac2408)

**Comment by mstreng411** [comment]
OMG - I'm so stupid, shame on me...
It is working now, thank you so much!!!
